frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -17,14 +17,21 @@ const { common } = Kebab.factory;
 export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
   item,
 }) => {
-  const { buildConfigs, hpas, routes, services, pods, obj } = item;
+  const { buildConfigs, hpas, routes, services, pods, obj, pipelines } = item;
   const hasBuildConfig = buildConfigs?.length > 0;
+  const hasPipeline = pipelines?.length > 0;
```

Comment:
Having pipelines as an indirect dependency here in core is bad because all pipeline knowledge should be controlled by the pipeline UI package (eventually separated to the operator).

Perhaps to simplify the code we can ask UX if we really need to _ever_ show the BuildOverview if there are no associated build configs? I'd much prefer controlling the visibility of the section based on whether or not there is any data to show.

Reply:
I added the check for pipeline since I didnt wanted to change the existing behavior in console where if a buildConfig is not present the buildSection is shown with an empty state until and unless a pipeline is associated and we do not need the buildSection any more as per our usecase. But if its okay to change this behavior I can simply just check for the buildConfig and show the section based on its availability.

Reply:
Already have their confirmation in the comments of the story.

[See my other comment on your PR](https://github.com/openshift/console/pull/6874/files#r510293235) for the link.

Reply:
Oh okay I thought it was just about devconsole. Will remove the check for pipeline.

Reply:
Done.